### PR TITLE
Break up wide tables and rename "Responsive/Autoscaling Policy" to "Controller Policy"

### PR DESCRIPTION
**Pull request by ableegoldman**
Also some minor typo fixes and formatting

### CODE REVIEW

File: docs/reference/scaling-policies.md [hunk 1]
```diff
@@ -88,32 +88,53 @@ properties:
...
+| spec.policyType           | The type of application under management. Must be set to KAFKA_STREAMS.                                                                                                        |
+| spec.kafkaStreamsPolicy   | Defines the policy for a Kafka Streams application.                                                                                                                            |
+
+## Kafka Streams Policies 
+
+Scope: `spec.kafkaStreamsPolicy`
+
+| Full Field Name                            | Description                                                                                                                                                                     |
+|--------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| spec.kafkaStreamsPolicy                    | Defines the policy for a Kafka Streams application.                                                                                                                             |
```
Comment (rodesai) [inline docs/reference/scaling-policies.md:112]: shall we drop the `spec.kafkaStreamsPolicy` prefix here?

File: docs/reference/scaling-policies.md [hunk 2]
```diff
@@ -88,32 +88,53 @@ properties:
...
 ```
 
-The following table defines what each of these fields means.
+The following tables define what each of these fields means.
+
+## Responsive Policy Specs 
+
+Scope: `spec`
+
+| Full Field Name           | Description                                                                                                                                                                    |
```
Comment (rodesai) [inline docs/reference/scaling-policies.md:97]: I think the word `Full` here and especially later is confusing if we're taking out the common prefixes

File: docs/reference/scaling-policies.md [hunk 3]
```diff
@@ -88,32 +88,53 @@ properties:
...
+| type                                                                                        | Specifies the type of this diagnoser. One of PROCESSING_RATE_SCALE_UP, PROCESSING_RATE_SCALE_DOWN, EXPECTED_LATENCY, or THREAD_SATURATION.                                                                                                                                                                                                                                              |
+| expectedLatency                                                                             | Specifies an Expected Latency Diagnoser. This Diagnoser specifies a goal for the expected latency for a record to be processed at a sub-topology, when enqueued at a source topic partition of that sub-topology.                                                                                                                                                                       |
```
Comment (rodesai) [inline docs/reference/scaling-policies.md:126]: we should probably split the different diagnosers into their own table (e.g. one for `expectedLatency`, one for `threadSaturation`, etc)

File: docs/reference/scaling-policies.md (hunk 2 above)
Comment (ableegoldman) [reply docs/reference/scaling-policies.md:97]: I guess the idea was to point out that we're listing the entire thing in the first two tables, but only the suffix with the common prefix/scope removed in the 3rd table. 

But maybe that's too confusing. I think I'll just remove the common prefix/scope from the 2nd table as well to keep the formatting consistent

File: docs/reference/scaling-policies.md (hunk 3 above)
Comment (ableegoldman) [reply docs/reference/scaling-policies.md:126]: good call, will do

**Review by rodesai — COMMENTED** [review]
Looks great! Thanks for the improvements.

**Pull request merged**